colossalai/zero/low_level/low_level_optim.py

```diff
@@ -892,3 +900,7 @@ def get_working_grad_by_param_id(self, param_id: int) -> Tensor:
     def get_partitioned_gradients_by_param_id(self, group_id: int, param_id: int) -> List:
         grad_store = self.pid_to_grad_store[param_id]
         return grad_store.get_partitioned_gradients_by_param_id(group_id, param_id)
+
+    def _force_waite_all_gather(self):
```

Comment:
What is the diff between `waite` and `wait`?

Reply:
typo